aikau/src/main/resources/alfresco/forms/controls/BaseFormControl.js

```diff
@@ -1439,6 +1439,38 @@ define(["dojo/_base/declare",
             this.alfLog("warn", "No watch method found on wrapped widget", this);
          }
       },
+
+      /**
+       * This is used to log whether or not this form control has had user focus or not. The reason being that in 
+       * certain circumstances we might not want to show validation errors until the user has at least made an attempt
+       * to enter a value before they are shown validation error messages. 
+       * 
+       * @instance
+       * @type {boolean}
+       * @default false
```

Comment:
Not necessary to specify the default for simple types ... just put `@default` and it'll auto-complete in the generated docs